pynicotine/pluginsystem.py

```diff
@@ -286,10 +286,10 @@ def plugin_settings(self, plugin_name, plugin):
                     plugin.settings[key] = customsettings[key]
 
                 else:
-                    log.add(_("Stored setting '%(name)s' is no longer present in the plugin"), {'name': key})
+                    log.add_debug(_("Stored setting '%(key)s' is no longer present in the '%(name)s' plugin"), {'key': key, 'name': plugin.__name__})
```

Comment:
This line is 149 characters long.

I'll be honest, i would prefer it if keys were deleted here instead. The only thing that would trigger this is if a developer modified the plugin and removed some plugin setting(s).

Reply:
Okay thanks, I will split the lines.

That is a good point, hopefully developers wont do this as it would make a mess of the user's config. However In the event of a key deletion, I think this log entry would even more desirable to keep it here (with slightly different wording).

Reply:
I don't follow your logic here i'm afraid.